apps/molgenis-components/src/components/forms/InputOntology.vue

```diff
@@ -87,6 +87,7 @@
           :key="key"
           :terms="rootTerms"
           :isMultiSelect="isMultiSelect"
+          :enableSelectNodes="!search"
```

Comment:
===

Reply:
huh? that doesn't make sense here this is a component assignment.

Reply:
Wait, did the code change? This wasn't where I thought I put this comment 😅